src/Illuminate/Console/Command.php

```diff
@@ -640,4 +632,46 @@ public function setLaravel($laravel)
     {
         $this->laravel = $laravel;
     }
+
+    /**
+     * Runs the console command.
+     *
+     * @param  \Symfony\Component\Console\Command\Command|string $command
+     * @param  array  $arguments
+     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
+     * @return int
+     */
+    private function runCommand($command, array $arguments, OutputInterface $output)
+    {
+        $arguments['command'] = $command;
+
+        return $this->resolveCommand($command)->run(
+            $this->createInputFromArguments($arguments), $output
+        );
+    }
+
+    /**
+     * Resolve console command instance.
+     *
+     * @param  \Symfony\Component\Console\Command\Command|string  $command
+     * @return \Symfony\Component\Console\Command\Command
+     */
+    private function resolveCommand($command)
+    {
+        if (! class_exists($command)) {
+            return $this->getApplication()->find($command);
+        }
+
+        $command = new $command;
```

Comment:
This makes #29666 fail, because a lot of commands use dependency injection.

Reply:
I've fixed this here: https://github.com/laravel/framework/pull/29666